recipes/ltl/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "ltl" %}
+{% set version = "2.0.19" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://www.as.utexas.edu/~drory/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f2f37220edd6aff55c9420745daf515e7f447b7a19a5f1250fe95d476384261
+  patches:
+    - 0001_Build_shared_lib.patch
+    - 0002_Fix_test_typo.patch
+    - 0003_Fix_blas_retun_type.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ name }}
```

Comment:
Should this constrain the version for the export, with something like `- {{ pin_subpackage(name, max_pin='x.x') }}`? Where the number of `x`s indicates the ABI stability

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements

Reply:
Good idea, I missed that. I just pushed a fixing commit.